[FEATURE] Add snippets for All and None (#4)
30-seconds/30-seconds-of-csharp: snippets/All.md

---
title: All

> tags: array,lambda,intermediate

Reviewer: ```suggestion
tags: array,lambda,overload,intermediate
```

---

Returns `True` if the provided predicate function returns `True` for all elements in a collection, `False` otherwise.

> Use `Array.TrueForAll()` to test if all elements in the collection return `True` based on `match`.

Reviewer: Maybe rephrase to:
```suggestion
Use `Array.TrueForAll()` to test if all elements in the collection return `True` based on the predicate function, `match`.
```

> Omit the second argument, `match`, to use the overload that checks if each value is different from `null` by default.

Reviewer: Maybe rephrase to:
```suggestion
Omit the predicate function, `match`, to use the overload that checks if each value is different from `null` by default.
```
